test/hw-device-context.js

```diff
@@ -0,0 +1,44 @@
+const os = require('bare-os')
+const test = require('brittle')
+const ffmpeg = require('..')
+const process = require('bare-process')
+
+if (os.platform() === 'darwin') {
+  test('HWDeviceContext should instantiate with device type', (t) => {
```

Comment:
```suggestion
  test('HWDeviceContext should instantiate with device type', { skip: os.platform() !== 'darwin' }, (t) => {
```

The same applies to the other cases; mark them skipped instead of conditionally declaring them.